frontend/src/Components/Dashboard/index.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import { Layout, Menu } from "antd";
+import {
+  UserOutlined,
+  LaptopOutlined,
+  NotificationOutlined,
+} from "@ant-design/icons";
+import "./index.css";
+
+import Profile from "../Profile";
+import Team from "../Team";
+import NotFound from "../NotFound";
+import { getUserID } from "../Firebase_funcs/firebase_funcs";
+
+const Dashboard = () => {
+  const { Content, Sider } = Layout;
+  const { SubMenu } = Menu;
+  const [collapsed, setCollapsed] = useState(false);
+  const userid = getUserID();
+
+  return (
+    <Router>
+      <Layout>
+        <Sider
+          theme="dark"
+          collapsible
+          collapsed={collapsed}
+          onCollapse={(collapsed) => setCollapsed(collapsed)}
+          width={250}
+          className="site-layout-background"
+        >
+          <Menu
+            theme="dark"
+            mode="inline"
+            defaultSelectedKeys={["0"]}
+            defaultOpenKeys={["sub1"]}
+            style={{ height: "100%", borderRight: 0 }}
+          >
+            <SubMenu key="sub1" icon={<UserOutlined />} title="Your teams">
+              <Menu.Item key="1">
```

Comment:
As szymon mentioned in other MR such repeated content can be done using and predefined array and a `map` function.
https://www.digitalocean.com/community/conceptual_articles/understanding-how-to-render-arrays-in-react

Reply:
I'll address this in #11.